### ISSUE TITLE
is this plugin usable ?

### ISSUE THREAD

**Issue by FuxonPlayz** [MCrafterss/MTeamPvP#26]
↑↑↑↑↑

**Comment by iPocket** [comment]
Not sure. we need testers, can you test it for us?

**Comment by MiroTheSuperDev** [comment]
lol  @iPocket we already have testers @MCrafterss/mtesters  pocketkiller needs you guys can you help

**Comment by iPocket** [comment]
@AmiroEmad I know, but they are not active...

**Comment by Sandertv** [comment]
I guess I could test it some time soon.

**Comment by FuxonPlayz** [comment]
I will test on my experiments server

**Comment by FuxonPlayz** [comment]
but can you add commands in plugin.yml ?

**Comment by iPocket** [comment]
No, Commands have been already added with a Command class, no need for command in plugin.yml

**Comment by iPocket** [comment]
Any bugs/results?

**Comment by MiroTheSuperDev** [comment]
should i test it my self for you instead of waiting for that guy

**Comment by iPocket** [comment]
@AmiroEmad Yes of course! Thanks

**Comment by MiroTheSuperDev** [comment]
ok i will do it shortly

**Comment by iPocket** [comment]
test

**Comment by iPocket** [comment]
Any testers please? I've been waiting along...

**Comment by AmiroEmad** [comment]
pocketkiller the plugin wouldnt generate any config untill a command is run you should add the add arena command so the plugin be able to use

**Comment by iPocket** [comment]
Of course!

**Comment by xZeroMCPE** [comment]
@iPocket Can't you test it? by yourself? instead of depending on people to do so? :+1:

**Comment by iPocket** [comment]
@xZeroMCPE How do you want me to test it without players and an port forwarded server?

**Comment by xZeroMCPE** [comment]
If you have multiple device. Join the same l local server  :)

**Comment by iPocket** [comment]
I don't.

**Comment by xZeroMCPE** [comment]
Oh ok, then we "do" need some testers!

**Comment by MiroTheSuperDev** [comment]
we have testers they are @MCrafterss/mtesters  but i think they are inactive most of them is

**Comment by FuxonPlayz** [comment]
hi I'm back
 I didn't receive any notifications from GitHub on my emai

anyway what is the command ?

**Comment by Sandertv** [comment]
/mtp is being used by MagicTelePortal too :/

**Comment by ghost** [comment]
There should be an custom (in config) command instead of /mtp, /pvp ect

**Comment by iPocket** [comment]
@ankitmaharjan252, I am doing it on a command class...
anyway, I will (maybe) do it on plugin.yml for others

**Comment by FuxonPlayz** [comment]
ok i will test it now!

**Comment by ghost** [comment]
Ops

**Comment by ghost** [comment]
sry i should ask before doing things cuz bored xD

**Comment by iPocket** [comment]
wait WHAT?

**Comment by iPocket** [comment]
Revert please!

**Comment by iPocket** [comment]
@Survingo thanks, but they mean the permissions of command etc.

**Comment by ghost** [comment]
No I won't revert because my github desktop has issues

**Comment by FuxonPlayz** [comment]
I typed /mtp /mteampvp /tpvp and nothing happen!

**Comment by FuxonPlayz** [comment]
I will try  Survingo new codes

**Comment by FuxonPlayz** [comment]
still not working!

**Comment by FuxonPlayz** [comment]
My Server is Running Genisys

**Comment by Ad5001** [comment]
Maybe try on clearsky which is less buggy and did not change the pm functions.

**Comment by MiroTheSuperDev** [comment]
dear @Ad5001 it works on genysis but pocketkiller made a sub command to work but he didnt code the main command the plugin just need more code but i am sure he will do it at a later time

**Comment by Ad5001** [comment]
Oh ok but I saw some errors on Genisys that don't happends on other third party builds (like entity spawning) so I wasn't sure about this.

**Comment by iPocket** [comment]
/mtp [create/delete] -_- Didn't you see the usage? Ok I'll make it user-friendly.

**Comment by iPocket** [comment]
Oh sorry, updated the comment, github markdown changed the <>..

**Comment by FuxonPlayz** [comment]
@iPocket I typed /mtp and no usage commands popped up 😑

**Comment by xZeroMCPE** [comment]
lol

**Comment by ghost** [comment]
xD

**Comment by iPocket** [comment]
@FuxonPlayz, Sorry my dumpness 0_0 Ok I updated it.

**Comment by MiroTheSuperDev** [comment]
@iPocket forgetting to code something or doing a mistake doesn't = dumpness 

most of the intelligent people who invented awesome and useful things did a lot of mistakes before they finished it that doesn't mean dumpness and that also doesn't mean they don't know what are they doing

**Comment by iPocket** [comment]
@AmiroEmad truth!

**Comment by iPocket** [comment]
No results yet 8 days please?

**Comment by MiroTheSuperDev** [comment]
my server have 0 players but i have lots of friends that i can call to play teampvp

**Comment by iPocket** [comment]
@AmiroEmad You meant 1, right? :P anyway, if you can port forward it, we may join to test there :)

**Comment by dc4code** [comment]
Lol

**Comment by dc4code** [comment]
I am working on something big.

http://vs3.dc4.us/FH2

**Comment by ghost** [comment]
Please contact me on skype if possible: dylan_c4
